prow/plank/reconciler.go

```diff
@@ -845,6 +846,10 @@ func isRequestError(err error) bool {
 	code := 500 // This is what kerrors.ReasonForError() defaults to.
 	if statusErr, ok := err.(kerrors.APIStatus); ok {
 		code = int(statusErr.Status().Code)
+	} else if wrappedErr := errors.Unwrap(err); wrappedErr != nil {
```

Comment:
Also check wrapped error, it's checked at https://github.com/kubernetes/test-infra/pull/23039/files#diff-f2aec52c49557c430579f10ddb0e1f5408f1ce88ef68e2daf687725e1d6846e7L265

Reply:
The whole passage can be simplified as:
```
	if status := APIStatus(nil); errors.As(err, &status) {
		code =  status.Status().Code
	}
	return metav1.StatusReasonUnknown
```

Which is in fact how the upstream matching supports wrapped errrors: https://github.com/kubernetes/kubernetes/blob/dadecb2c8932fd28de9dfb94edbc7bdac7d0d28f/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go#L649-L652

Reply:
good to know, done.